docs/metrics/health-checks.md

```diff
@@ -0,0 +1,54 @@
+---
+title: Data Health Checks
+sidebar_label: Health
+slug: /metrics/health-checks
+---
+
+### Monitoring Data Health
+
+To monitor the health of your data,
+
+- Navigate to **Metrics** in the left-hand navigation panel in the Statsig console
+- Select the **events** tab at the top
+
+- **Data Health Checks Card**
+
+  ![Screen Shot 1402-04-06 at 15 31 53](https://github.com/statsig-io/docs/assets/133702265/07997263-9840-4e09-ae2f-e709e852df71)
+
+  - The data health checks card displays the results of the health checks that are run on your data. If the data passes all the health checks, the card is collapsed by default. If any health check fails, the card is automatically expanded revealing the health check results.
```

Comment:
delete: If the data passes all the health checks, the card is collapsed by default.